bundles/org.openhab.ui/web/src/components/cards/glance/location/status-badge.vue

```diff
@@ -70,6 +71,10 @@ export default {
     query () {
       let direct, equipment, allPoints, points
       switch (this.type) {
+        case 'battery':
+          direct = findPoints(this.element.properties, 'Point_Status_LowBattery', true)
+          if (direct.length) return direct
+          return findPoints(allEquipmentPoints(this.element.equipment), 'Point_LowBattery', true)
```

Comment:
You missed this one!

```suggestion
          return findPoints(allEquipmentPoints(this.element.equipment), 'Point_Status_LowBattery', true)
```

Reply:
🤦🏼 Ups ... changed it.